.circleci/build-and-test/workflows.yml

```diff
@@ -3,15 +3,9 @@
     when: << pipeline.parameters.build_and_test_all >>
     jobs:
       - secrets-check
-      - test-backend:
-          requires:
-            - secrets-check      
-      - test-frontend:
-          requires:
-            - secrets-check
-      - test-e2e:
-          requires:
-            - secrets-check
+      # TODO: Revert back to tests after review
```

Comment:
@elipe17 this PR is ready to revert changes needed for testing. build-and-test-all workflow was successful. Please ping me for qasp approval after the revert. 

![zaphotfix](https://github.com/raft-tech/TANF-app/assets/63075587/c20b9b2b-7adf-4128-9592-05097e6d626a)

Reply:
@ADPennington I have reverted the workflow to it's original state